core/src/components/datetime/datetime.tsx

```diff
@@ -1723,6 +1718,7 @@ export class Datetime implements ComponentInterface {
 
     return (
       <ion-picker-column-internal
+        class="hour-column"
```

Comment:
Is there a reason we introduced class names for: `hour-column`, `minute-column` and `day-period-column`?

Reply:
I added those for some old tests, but those tests are no longer used. I will remove them.